ext/civiimport/civiimport.php

```diff
@@ -118,7 +118,7 @@ function _civiimport_civicrm_get_import_tables(): array {
       'user_job_id' => (int) $tables->id,
       'created_date' => $tables->created_date,
       'expires_date' => $tables->expires_date,
-      'title' => ts('Import Job') . (int) $tables->id,
+      'title' => ts('Import Job') . $tables->id,
```

Comment:
Should there be a space in there, and/or using a placeholder? And also this is ts but the one above is E::ts.

Reply:
Yea good point @demeritcowboy 
See #26593